src/board/__tests__/board.test.tsx

```diff
@@ -165,48 +124,57 @@ describe("Board", () => {
     expect(container.ownerDocument.body).not.toHaveClass(resizeClass);
   });
 
+  test("applies pointer interaction class name", () => {
+    const { container } = render(<Board {...defaultProps} />);
+
+    const disableSelectionClass = globalStateStyles["disable-selection"];
+    expect(container.ownerDocument.body).not.toHaveClass(disableSelectionClass);
+
+    const handle = createWrapper().findBoardItem()!.findDragHandle()!.getElement();
+
+    fireEvent(handle, new MouseEvent("pointerdown", { bubbles: true }));
+    expect(container.ownerDocument.body).toHaveClass(disableSelectionClass);
+
+    fireEvent(window, new MouseEvent("pointerup", { bubbles: true }));
+    expect(container.ownerDocument.body).not.toHaveClass(disableSelectionClass);
+  });
+
+  test("does not apply pointer class when keyboard is used", () => {
+    const { container } = render(<Board {...defaultProps} />);
+    const dragHandle = createWrapper().findBoardItem()!.findDragHandle()!;
+    const disableSelectionClass = globalStateStyles["disable-selection"];
+
+    dragHandle.keydown(KeyCode.enter);
+    expect(container.ownerDocument.body).not.toHaveClass(disableSelectionClass);
+
+    dragHandle.keydown(KeyCode.down);
+    expect(container.ownerDocument.body).not.toHaveClass(disableSelectionClass);
+
+    dragHandle.keydown(KeyCode.enter);
+    expect(container.ownerDocument.body).not.toHaveClass(disableSelectionClass);
+  });
+
   test("sets active state for drag handle", () => {
-    render(
-      <Board
-        items={[
-          { id: "1", data: { title: "Item 1" } },
-          { id: "2", data: { title: "Item 2" } },
-        ]}
-        renderItem={(item) => <BoardItem i18nStrings={itemI18nStrings}>{item.data.title}</BoardItem>}
-        onItemsChange={() => undefined}
-        i18nStrings={i18nStrings}
-        empty="No items"
-      />
-    );
+    render(<Board {...defaultProps} />);
 
-    const dragHandle = createWrapper().findBoardItem()!.findDragHandle()!.getElement();
-    const resizeHandle = createWrapper().findBoardItem()!.findResizeHandle()!.getElement();
+    const findBoardItem = () => createWrapper().findBoard()!.findItemById("1")!;
+    const findDragHandle = () => findBoardItem().findDragHandle().getElement();
+    const findResizeHandle = () => findBoardItem().findResizeHandle().getElement();
 
-    expect(dragHandle).not.toHaveClass(dragHandleStyles.active);
+    expect(findDragHandle()).not.toHaveClass(dragHandleStyles.active);
```

Comment:
Because of using portals, `dragHandle` is not the same node before drag and during the drag (React does not move nodes across portals, https://github.com/facebook/react/issues/13044). 

Calling a function to fetch the actual element after every interation